atlas/yarn.lock

```diff
@@ -16994,7 +16983,7 @@ redeyed@~2.1.0:
   dependencies:
     esprima "~4.0.0"
 
-redux@*, redux@^4.0.0, redux@^4.0.5:
+redux@*, redux@^4.0.0:
```

Comment:
There's still `redux` left here? Does some other dependency rely on it?

Reply:
There were `@types/redux` and `@types/react-redux` left in our dependencies, removed both